src/services/azureFunctionsService.ts

```diff
@@ -68,7 +68,25 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 		}
 		let projectFile = await azureFunctionUtils.getAzureFunctionProject();
 		if (!projectFile) {
-			vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened);
+			let projectCreate = await vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened, constants.learnMore, constants.createProject);
+			if (projectCreate === constants.learnMore) {
+				vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(constants.sqlBindingsDoc));
+			} else if (projectCreate === constants.createProject) {
+				// ask what folder the project should be created in
+				const options: vscode.OpenDialogOptions = {
```

Comment:
I'm not sure we need to prompt here - the create flow will already do that so might as well let them handle the whole thing.